proarc-common/src/main/java/cz/cas/lib/proarc/common/export/mets/structure/MetsElementVisitor.java

```diff
@@ -190,16 +191,31 @@ protected void initHeader(IMetsElement metsElement) {
         MetsHdr metsHdr = new MetsHdr();
         metsHdr.setCREATEDATE(metsElement.getCreateDate());
         metsHdr.setLASTMODDATE(metsElement.getLastUpdateDate());
-        Agent agent = new Agent();
-        agent.setName(metsElement.getMetsContext().getCreatorOrganization());
-        agent.setROLE("CREATOR");
-        agent.setTYPE("ORGANIZATION");
-        metsHdr.getAgent().add(agent);
+
+        setAgent(metsHdr, "CREATOR", "ORGANIZATION", metsElement.getMetsContext().getOptions().getCreator());
```

Comment:
Alespoň jedno? Já vidím, že chyba se vyhodí v obou případech a podle spec je to správně. Jen by tam mělo být jasně napsané co je potřeba opravit. Název nastavení.